pregred-gui/src/app/pages/home/Home.js

```diff
@@ -1,9 +1,255 @@
-import React from 'react';
+import React,{useState} from 'react';
 
 import './Home.css';
+import { Modal, Button } from 'antd';
 
 function Home() {
-  return <h1> PregRed App is Running - Home </h1>;
+  const [visible, setVisible1] = useState(false);
+  const [visible2, setVisible2] = useState(false);
+  const [visible3, setVisible3] = useState(false);
+  const [visible4, setVisible4] = useState(false);
+  const [visible5, setVisible5] = useState(false);
+  const [visible6, setVisible6] = useState(false);
+  const [visible7, setVisible7] = useState(false);
+  return(
+    <div className="container-home">
+
+         <div className="container-home-div1">
+           {/* <h3 className="homepage-para-header">Abstract</h3> */}
+            <p className="homepage-para">
+            The pregnancy period is one of the most important periods in the life of a woman.
+            The health of the baby is the most crucial factor in that period.
+            <br/>
+            PregRed will help you to detect
+            how the pre-pregnancy factors affect the baby’s health,
+            get knowledge about pre-pregnancy health impacts and can meet health
+            professionals.
+            </p>
+
+{/* Form 1 */}
+        <div className="home-main-div-models">
+            <div className="home-div-models">
+              <Button type="primary" shape="round" onClick={() => setVisible1(true)}>
+                Why ladies should aware of prepregnancy health care?
+              </Button>
+              <Modal
+                title="Why ladies should aware of prepregnancy health care?"
+                centered
+                visible={visible}
+                onOk={() => setVisible1(false)}
```

Comment:
if both ok and cancel do the same work, can remove cancel here and similar modals.

Reply:
I tried to remove it. But I couldn't find it because it already formed from antd.